src/Altinn.App.Api/Models/InstanceDto.cs

```diff
@@ -0,0 +1,189 @@
+#nullable disable
+using Altinn.Platform.Register.Models;
+using Altinn.Platform.Storage.Interface.Models;
+using Newtonsoft.Json;
+
+namespace Altinn.App.Api.Models;
+
+/// <summary>
+/// Represents the response from an API endpoint providing a list of key-value properties.
+/// </summary>
+[JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
```

Comment:
Do we need to copy the Newtonsoft annotations over here?

Reply:
Nope I don't think so, I can remove them